src/converters/lintConfigs/rules/convertRules.test.ts

```diff
@@ -353,6 +353,45 @@ describe("convertRules", () => {
         ]).toContainEqual(converted);
     });
 
+    it("merges when a rule's ruleSeverity explicitly differs from its TSLint equivalent", () => {
```

Comment:
This test case looks good and we should include it. Great!

But I think we should also test for the case described in the issue: that there are _two_ output rules from the converter, one turned off and one with the original rule severity. Could you please add a test case for that?

Reply:
For sure! I'll incorporate that new test case in my upcoming modifications ^w^